docs/faqs/rileylink-faqs.md

```diff
@@ -1,67 +1,67 @@
-# Radio-Link FAQs
+# RileyLink Compatible Device FAQs
 
-## What is the radio-link? Do I have to have one?
+## What is the RileyLink compatible device? Do I have to have one?
 
-The radio-link is a required part of Loop. The radio-link is a little device that helps your iPhone and pump/pod speak to each other. The radio-link is a critical part of the Loop system because the pump needs communications via radio-frequency, and the iPhone needs communications via Bluetooth. The radio-link is like a translator that can speak both formats so that the pump and iPhone can loop together. The radio-link communicates information to/from your pump by radio communications, and to/from your iPhone using Bluetooth.
+The RileyLink compatible device is a required part of Loop. It is a small device that helps your iPhone and pump/pod speak to each other. The Loop compatible pumps communicate via radio-frequency and the iPhone communicates via Bluetooth. The device is like a translator that can speak both formats so that the pump and iPhone can loop together. The device communicates information to/from your pump by radio communications, and to/from your iPhone using Bluetooth.
```

Comment:
Perhaps this could be augmented by mentioning the "RileyLink protocol" here. For example, instead of just "to/from your iPhobe using Bluetooth"  say "using the RileyLink protocol over Bluetooth".